rnn_serving/rnn_serving.ipynb

```diff
@@ -0,0 +1,236 @@
+{
```

Comment:
<p>Then use the wasabi url to add the model file in <code>fn.add_model</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/functions/pull/110/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>